[terraform] Add GCP persistent disk (#78)
bitergia-analytics/bap-deployment-toolkit: docs/provision.md

If you have doubts about how to list the users, please check the following
[link](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/iap_tunnel_instance_iam#argument-reference).

> All instaces create by default a boot disk with the same name as the instance,
> to make it persistent you have to set persistent_disks to true.

Reviewer: ```suggestion
By default, all instances will create a boot disk with the same name of the instance.
To make it persistent, you will have to set `persistent_disks` to true.
```

- `persistent_disks = true`.

> You can also create an extra persistent disk that you can attach to any
> VMs by default it will attach to the same name VM. If disk count is not set or
> the value is `0` it will use the node count value.

Reviewer: ```suggestion
You can also create an extra persistent disk that can be attached to any VMs.
It will be attached to the same name of VM, by default. When the `<node>_disk_count`
is not set or the value is `0`, it will use the `<node>_count` value.
```

- `mariadb_disk_count = 2`
- `mariadb_disk_attach = "<VM-NAME>"`

> There is an option to create a disk from a snapshot.

Reviewer: ```suggestion
You can also create a disk from a snapshot.
```
